pkg/operator/ceph/disruption/clusterdisruption/pools.go

```diff
@@ -129,10 +129,15 @@ func (r *ReconcileClusterDisruption) getOngoingDrains(request reconcile.Request)
 
 			nodeList := &corev1.NodeList{}
 			err = r.client.List(context.TODO(), nodeList, client.MatchingLabels{corev1.LabelHostname: nodeHostname})
-			nodeNum := len(nodeList.Items)
-			if err != nil || nodeNum < 1 {
+			if err != nil {
```

Comment:
@travisn moved the error checking up!

Reply:
If the node doesn't exist I believe you will return an error here instead of entering the `else` statement to delete it on lines 135-136. What about this? 
```
if err != nil {
  if errors.IsNotFound(err) {
     // handle node not found
  }
  // return the unknown error
}

Reply:
with the  `List` method, `errors.IsNotFound(err)` doesn't really make sense. It will just return an empty list.

Reply:
oh right, `List()` wouldn't return a not found error. my bad